.github/workflows/cicd.yaml

```diff
@@ -460,19 +460,26 @@ jobs:
         run: |
           docker run --rm --name postgres-db -p 5432:5432 -e POSTGRES_PASSWORD=postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=${{ needs.setup_variables.outputs.app }} -d postgres:13.13
 
-      - uses: shivammathur/setup-php@2.30.2
+      - name: authenticate to composer
+        run: |
+          echo "{\"github-oauth\": {\"github.com\": \"${{secrets.numa-numa-token}}\"}}" > $GITHUB_WORKSPACE/auth.json
+
+#      Workaround failing GH token in composer (Telecom/Vehicle service)
+      - name: Add SSH key
+        uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.gh-ssh-key }}
+
+      - uses: shivammathur/setup-php@v2
         if: inputs.ApplicationLanguage == 'php'
         with:
           php-version: ${{inputs.PhpVersion}}
           extensions: ${{inputs.PhpExtensions}}
           coverage: pcov
           tools: composer
         env:
-          runner: ${{needs.setup_variables.outputs.runners}}
+          runner: ubuntu-latest
```

Comment:
this should stay as ${{needs.setup_variables.outputs.runners}} - its already "hard set" to ubuntu-latest on line #220 but it gives us a way to quickly change it